impl/console.cxx

```diff
@@ -6,12 +6,15 @@
 #ifndef _WINDOWS
 #	include <unistd.h>
 #else
+#	ifndef NOMINMAX
+#	define NOMINMAX
+#	endif
+#	ifndef WIN32_LEAN_AND_MEAN
 #	define WIN32_LEAN_AND_MEAN
+#	endif
 #	include <windows.h>
-#	include <stringapiset.h>
```

Comment:
We seem to recall this used not to be included in `<windows.h>`, guessing this changed?

Reply:
Unsure, it's detailed as such here: https://learn.microsoft.com/en-us/windows/win32/api/stringapiset/nf-stringapiset-multibytetowidechar#requirements

Though it looks to be with very old mingw-w64 headers: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/837

Reply:
Ah, thank you, most excellent! We'll merge this then.